src/store/auth-context.js

```diff
@@ -42,6 +43,7 @@ const AuthProvider = ({ children }) => {
   const logoutHandler = () => {
     setIsLoggedIn(false);
     localStorage.removeItem("token");
+    localStorage.removeItem("user");
     navigate("/");
```

Comment:
You can extract "token" and "user" as constants

Reply:
ok will make the changes